i18n/en.pot

```diff
@@ -636,8 +639,8 @@ msgstr "Group"
 msgid "Filters"
 msgstr "Filters"
 
-msgid "Failed to load layer \"{{layername}}\": {{message}}"
-msgstr "Failed to load layer \"{{layername}}\": {{message}}"
+msgid "{{layername}}: {{message}}"
```

Comment:
```suggestion
msgid "{{- layername}}: {{- message}}"
```
is the escaping needed or could it be an issue if the strings contain special characters?

Reply:
Good point. The message is always the same, so that can be hardcoded. The layername will change though.

Reply:
Ok, if the `layername` is considered "safe" then I suggest adding `{{- layername}}` to avoid potential escaping issues in the future